ledger/src/use_snapshot_archives_at_startup.rs

```diff
@@ -10,9 +10,13 @@ pub enum UseSnapshotArchivesAtStartup {
     /// already on disk.  This will incur the associated runtime costs for extracting.
     #[default]
     Always,
-    /// If snapshot archive are not used, then the local snapshot state already on disk is
+    /// If snapshot archives are not used, then the local snapshot state already on disk is
     /// used instead.  If there is no local state on disk, startup will fail.
     Never,
+    /// Only use snapshot archives if they are newer than the local snapshot state on disk.
```

Comment:
should we clarify that if there is no local snapshot state it will also use archives?

Reply:
I don't think that requires a name-change of the variant.